src/modules/flow/int/int.c

```diff
@@ -451,12 +451,17 @@ irange_constrain_open(struct sol_flow_node *node, void *data, const struct sol_f
     return 0;
 }
 
-static void
+static int
 irange_constrain(struct sol_irange *value)
 {
+    if(value->step == 0) {
```

Comment:
Please use the `check-style` tool prior to submitting a patch.

Also -- instead of making this an error, why not just ignore `value->step` when it's zero? The value will be constrained nonetheless:

    if (step)
        val -= (val - min) % step;
    if (val < min) val = min;
    if (val > max) val = max;